erdpy/transactions.py

```diff
@@ -49,7 +49,7 @@ def __init__(self):
         self.options = 0
         self.signature = ""
 
-    # The data field is base64-encoded. erdpy only supports utf-8 "data" at this moment.
+    # The data field is base64-encoded. mxpy only supports utf-8 "data" at this moment.
```

Comment:
We can say something like `Here, we only support ...` (no need to say `mxpy` everytime).